src/oc_erchef/apps/chef_objects/test/chef_user_tests.erl

```diff
@@ -180,6 +180,26 @@ parse_binary_json_tests(Version) ->
                 ?assertEqual(lists:sort(MinValid), lists:sort(GotData))
       end
      },
+     {?VD("Can create user with empty middle name"),
```

Comment:
In addition to these unit tests, we will want to add oc-chef-pedant test cases for the /users API.